src/shared/compatibility.js

```diff
@@ -1812,6 +1812,14 @@ PDFJS.compatibilityChecked = true;
         case 'javascript':
         case 'mailto':
           return 'null';
+        case 'blob':
+          // Special case of blob: -- returns valid origin of _schemaData.
+          try {
+            return new JURL(this._schemaData).origin || 'null';
```

Comment:
Don't you mean this._schemeData here, since the above property is always undefined (applies in the comment below as well)!?